sdk/spring/spring-cloud-azure-appconfiguration-config/src/main/java/com/azure/spring/cloud/appconfiguration/config/implementation/JsonConfigurationParser.java

```diff
@@ -44,10 +45,16 @@ static boolean isJsonContentType(String contentType) {
     }
 
     static Map<String, Object> parseJsonSetting(ConfigurationSetting setting)
-        throws JsonProcessingException {
+        throws IOException {
         Map<String, Object> settings = new HashMap<>();
-        JsonNode json = MAPPER.readTree(setting.getValue());
-        parseSetting(setting.getKey(), json, settings);
+        try {
+            JsonNode json = MAPPER.readTree(setting.getValue());
+            parseSetting(setting.getKey(), json, settings);
+        } catch (JsonProcessingException e) {
+            throw new IOException(
```

Comment:
InvalidConfigurationPropertyValueException

Reply:
I checked the constructor of `InvalidConfigurationPropertyValueException`. It requires `name`, `value` and `reason`. 
What shall we provide for `value`? We didn't want to log the value

Reply:
If you read the description a null value for the `value` field is allowed.

Reply:
But is it a little wired that if we use a `null` value? 
The error message would be `Property test_key with value 'null' is invalid: Expected type: JSON String, Number, Array, Object or token 'null', 'true' or 'false'`

Besides, I also tried to pass an empty string as value. It's also a little wired.
`Property test_key with value '' is invalid: Expected type: JSON String, Number, Array, Object or token 'null', 'true' or 'false'`

Reply:
Maybe we do what we do with our logs and put `<Redacted>`

Reply:
Do you mean we still use `null` when throw `InvalidConfigurationPropertyValueException`. But when logging this error, we replace the `null` string to other string?